### DISCUSSION TITLE
Can  variables in the Workspace add a search function？

### DISCUSSION THREAD

**Discussion by wilikeit** [pyzo/pyzo#1079] [Ideas]
Just like spyder, you can search for variables.If there are more variables, the search will be easier

![image](https://github.com/user-attachments/assets/eae16e0d-071c-423a-a0da-a96b6c7d35f8)

**Comment by bdieterm** [comment]
In the current main branch there is already a kind of search function for variable names. See #1057 and #1060.
To use that feature you can either run Pyzo from source or replace the pyzoWorkspace.py file in your current Pyzo directory with the new one:
https://github.com/pyzo/pyzo/blob/main/pyzo/tools/pyzoWorkspace.py
Otherwise you have to wait for the next release.

**Comment by wilikeit** [comment]
Thanks, this should be a nice feature to use.
